Header header

string status
bool converged
bool degenerate
float32 optimization_time
int32 iteration_count
int32 iteration_max
float32 translation_residual
float32 rotation_residual